summary_workflows/identification/I2_benchmark/I2_benchmark_specification.md

```diff
@@ -47,45 +59,128 @@ Fields:
 - **score** - not used, leave as "."
 - **strand** - defines the strand; either "." (=no strand) or "+" or "-".
 
-#### Format 2
+#### Input 2
 
 This BED file contains single-nucleotide position of poly(A) sites identified from the orthogonal 3'end-seq dataset.  
-Fields are the same as in Format 1.
+Fields are the same as in Input 1.
 	
+## Plots
+
+The results of this benchmark will be visualised in OpenEBench using the following plots:
+
+1. **bar plot** visualizing **Precision** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
+
+- 10 nt
+- 50 nt
+- 100 nt
+
+**X axis** - benchmarked tool  
+**Y axis** - precision
+
+Input datasets:
+
+- RNA-Seq data compared with 3'end sequencing data
+- Simulated RNA-Seq data compared with dataset used for simulation
+
+
+Ranking: The best performing tool is the one with the highest precision value.
+
+2. **2D scatter plot** visualizing **TPR and FPR** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
+
+- 10 nt
+- 50 nt
+- 100 nt
+
+**X axis** - FPR(d)  
+**Y axis** - TPR(d)  
+where _d - distance threshold_
```

Comment:
There would be 3 separate plots, each for a different distance threshold, with one point per participant showing FPR on X axis and TPR on Y axis. I don't think 3 points per participant in one plot can be reasonably shown in OEB, but if they can (e.g. same color but different shape), it would be great to decrease the number of plots.